frontend/src/app/sessions/active-sessions/active-sessions.component.html

```diff
@@ -54,9 +54,7 @@ <h3>{{ sessionService.beatifyState(session.state).text }}</h3>
           mat-flat-button
           color="primary"
           (click)="openReconnectDialog(session)"
-          [disabled]="
-            !(session.state === 'Started' || session.state === 'START_SESSION')
-          "
+          [disabled]="!sessionService.beatifyState(session.state).success"
```

Comment:
I am not sure if this function should really be called `beatify` and not `beautify`. Also, there is a similar function in easebackup.service.ts that should be deleted if it is deprecated, or the function names should be updated to distinguish between the two.

Reply:
I have renamed it. I'll rework the whole EASE Backup component in another PR, I'll get rid of it there.